requirements.txt

```diff
@@ -117,7 +117,7 @@ invenio-config==1.0.0
 invenio-csl-rest==1.0.0a1
 invenio-db==1.0.0
 invenio-deposit==1.0.0a9
-invenio-files-rest==1.0.0a23
+invenio-files-rest==1.0.0a23.post1
```

Comment:
Requirements.txt should be kept on par with setup.py, so implement the change there as well :)